packages/flutter/test/material/app_bar_theme_test.dart

```diff
@@ -490,9 +490,11 @@ void main() {
     }
 
     await tester.pumpWidget(buildFrame(appIconColor: Colors.lime));
+    await tester.pumpAndSettle();
```

Comment:
Is this change because of the newly added `AnimatedTheme` in `ButtonStyleButton`?

Reply:
Yes, Themes requires `tester.pumpAndSettle` to rebuild the widget with updated theme.

Reply:
In my experience, when testing widget parameter and theme parameter in the same tests always requires `tester.pumpAndSettle`. These tests are doing the same.